docs/SOPs/GDPR_Guidelines.md

```diff
@@ -18,6 +18,18 @@ parent: SOPs
 
 This document will serve to display our guidelines for GDPR and data protection restrictions when wrangling. 
 
+## Summary diagram
+
 <iframe width="768" height="432" src="https://miro.com/app/live-embed/o9J_lzQKwKI=/?moveToViewport=-2604,-1179,5558,2621" frameBorder="0" scrolling="no" allowFullScreen></iframe>
 
 ![image](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/assets/images/GDPR_screenshots/GDPR_Flowchart.jpg?raw=true)
+
+## Specific guidelines
+
+GDPR concers arise when dealing with living donor organisms, as HCA sticks to a strict interpretation of the regulations. When wrangling any kind of data related to living donors, wranglers can only ingest into the system data that is already in the public domain.
+
```

Comment:
I suggest changing this to:

"When wrangling data related to living donors, wranglers can only ingest into the system data that is already in the public domain." and then specifying what data types this includes e.g.:

"Here, by data we specifically refer to:

-  raw fastq files
- raw gene expression matrix (TBD - discussion with Tony)
- processed gene expression matrix (TBD - discussion with Tony) "

Wranglers should not ask contributors for this extra data if it is not already present in the public domain.

Reply:
Added this